### PR TITLE
Adding NFT balance command

### PR DESCRIPTION
**Pull request by aaitor**
## Description

* balance command
* clone command
* adjustment to contracts v3.2.1

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: resources/commands.json [hunk 1]
```diff
@@ -983,6 +983,31 @@
...
+				"positionalArguments": [{
+					"name": "nftAddress",
+					"type": "string",
+					"description": "The NFT contract address"
+				}],
+				"optionalArguments": [{
+						"name": "address",
+						"type": "string",
+						"default": "",
+						"description": "The address of the account to get the balance"
```
Comment (r-marques) [inline resources/commands.json:1001]: should this be an optional argument?

File: resources/commands.json [hunk 2]
```diff
@@ -1396,6 +1421,34 @@
 						"description": "The NFT type"
 					}
 				]
+			}, {
+				"name": "balance [nftAddress] [did]",
+				"description": "It gets the balance of a NFT",
+				"details": "This command allows to check if an address is a holder of a specific NFT wrapped by a Nevermined DID",
+				"examples": ["ncli nfts1155 hold did:nv:afd733c23c41af948be7ec039c3fb2048d437e082a69ea3f336cdf452a49be7e"],
```
Comment (r-marques) [inline resources/commands.json]: this example does not match with the command `balance` vs `nfts 1155 hold`

**Review by r-marques — APPROVED** [review]

File: resources/commands.json (hunk 1 above)
Comment (aaitor) [reply resources/commands.json:1001]: I think so because if is not given it try  to get it from the DDO

File: resources/commands.json (hunk 2 above)
Comment (aaitor) [reply resources/commands.json]: fixed

**Pull request merged**